src/compiler.ts

```diff
@@ -3520,9 +3520,12 @@ export class Compiler extends DiagnosticEmitter {
   ): ExpressionRef {
     var module = this.module;
 
-    // void to any
     if (fromType.kind == TypeKind.VOID) {
-      assert(toType.kind != TypeKind.VOID); // convertExpression should not be called with void -> void
+      if (toType.kind == TypeKind.VOID) {
+        // void to void
```

Comment:
```suggestion
        // void to void: Can happen as a result of a foregoing error. Since we
        // have an `expr` here that is already supposed to be void, return it.
```